src/internal/pfsdb/branches.go

```diff
@@ -474,7 +520,52 @@ func CreateDirectBranchProvenance(ctx context.Context, ext sqlx.ExtContext, from
 	return nil
 }
 
+// GetTriggerBranches lists all the branches related to a branch that trigger it and are triggered by it
+func GetTriggerBranches(ctx context.Context, ext sqlx.ExtContext, bid BranchID) (triggered []*pfs.Branch, triggering []*pfs.Branch, retErr error) {
```

Comment:
I wonder if there is value to splitting this into two functions (since its two queries)?